recipes/perl-sub-name/build.sh

```diff
@@ -0,0 +1,19 @@
+
+set -o errexit -o pipefail
```

Comment:
This one misses the shebang line and has end-of-file whitespace, but let's deal with that after merging.